src/Init/System/IO.lean

```diff
@@ -220,7 +220,7 @@ local macro "nonempty_list" : tactic =>
 @[extern "lean_io_get_num_heartbeats"] opaque getNumHeartbeats : BaseIO Nat
 
 inductive FS.Mode where
-  | read | write | readWrite | append
+  | read | write | writeNew | readWrite | append
```

Comment:
This needs a clarifying docstring, in which case we probably should clarify all of them.